src/target/mod.rs

```diff
@@ -665,6 +665,13 @@ pub trait Target {
     fn support_auxv(&mut self) -> Option<ext::auxv::AuxvOps<'_, Self>> {
         None
     }
+
+    /// Support for querying register info in LLDB.
+    /// Let the debugger request register info one by one. Should be used if 'use_target_description_xml()' is disabled.
```

Comment:
nit: add a newline between the brief text and the expanded text, or else rustdoc will put everything on one line.